itou/templates/siaes/edit_job_description.html

```diff
@@ -0,0 +1,67 @@
+{% extends "layout/content_small.html" %}
+{% load static %}
+{% load bootstrap4 %}
+
+{% block title %}Enregistrer une fiche de poste{{ block.super }}{% endblock %}
+
+{% block breadcrumbs %}
+    {% include "siaes/includes/_job_description_breadcrumbs.html" %}
+{% endblock %}
+
+{% block content %}
+    <div class="c-stepper my-3 my-lg-6">
+        <div class="progress progress--emploi">
+            <div class="progress-bar" role="progressbar" style="width: 33%" aria-valuenow="33" aria-valuemin="0" aria-valuemax="100"></div>
+        </div>
+        <p><strong>Étape 1</strong>/3 : Informations générales</p>
+    </div>
+
+    <div class="d-block rounded-lg shadow-lg p-3 p-lg-5">
+        <div class="text-center mb-3">
+            <img src="{% static 'img/edit_job_description_header.svg' %}" alt="En-tête édition fiche de poste" style="width: 100px;">
```

Comment:
```suggestion
<img src="{% static 'img/edit_job_description_header.svg' %}" alt="" class="w-100">
```
L'image étant purement décorative, je ne pense pas que ce soit nécessaire d'en faire lire la description aux lecteurs d'écran.